.gitmodules

```diff
@@ -51,6 +51,7 @@
 [submodule "vendor/vast/src"]
 	path = vendor/vast/src
 	url = https://github.com/trailofbits/vast.git
+	branch = mx_codegen
```

Comment:
Is `mx_codegen` branch is maintained? Why not built with `master` branch and merge required changes from `mx_codegen`?

Reply:
That will require multiplier to switch to llvm 19.

Reply:
I made a brief attempt in 2024 to update to latest VAST, but there were a lot of issues because it started using C++ ranges heavily in a few places, and the Clang patches for PASTA/Multiplier patches fell over on them. There were other issues, even after commenting out the relevant code in VAST. Eventually I got too busy with Patchestry and gave up. I think the proper solution in the future is to re-do all or parts of Multiplier's bootstrap, but using "plain old Clang", rather than PASTA. It's also possible that there's a simple fix. Something about PASTA's Run.cpp is potentiall off.